services/galley/src/Galley/API/Clients.hs

```diff
@@ -10,16 +10,17 @@ module Galley.API.Clients
 
 import Data.Id
 import Galley.App
-import Galley.Types.Clients (clientIds)
+import Galley.Types.Clients (clientIds, fromUserClients)
 import Network.Wai
 import Network.Wai.Predicate hiding (setStatus)
 import Network.Wai.Utilities
 
-import qualified Galley.Data as Data
+import qualified Galley.Data         as Data
+import qualified Galley.Intra.Client as Intra
 
 getClients :: UserId -> Galley Response
 getClients usr = do
-    clts <- Data.lookupClients [usr]
+    clts <- fromUserClients <$> Intra.lookupClients [usr]
```

Comment:
This is not the only place where this is used, see
```
src/Galley/API/Update.hs
442:    clts  <- Data.lookupClients users
461:    clts  <- Data.lookupClients (map memId membs)
```
(i.e. for every message sent, clients are looked up)

Please, for all three cases, add a configuration option allowing to use the cassandra-galley or the brig lookup depending on a configuration value.

Reply:
sorry, I had actually seen those usages and forgot to replace them. I will add it with the configuration options.